Petko/Zad2.cpp

```diff
@@ -0,0 +1,76 @@
+// ConsoleApplication1.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+struct OrderedPair {
+	int first;
+	int second;
+};
+
+void inputPair(OrderedPair& p)
+{
+	cin >> p.first >> p.second;
+}
+bool compairPairs(const OrderedPair& p, const OrderedPair& p1)
+{
+	return (p.first > p1.first || ((p.first == p1.first) && (p.second > p1.second)));
+}
+void biggestPair(const OrderedPair* arr, int& maxFirst, int& maxSecond)
+{
+	maxFirst = arr[0].first;
+	maxSecond = arr[0].second;
+	for (size_t i = 1; i < 100; i++)
+	{
+		if (arr[i].first > maxFirst && arr[i].second > maxSecond)
+		{
+			maxFirst = arr[i].first;
+			maxSecond = arr[i].second;
+		}
+	}
+}
+void swap(OrderedPair& first, OrderedPair& second)

```

Comment:
There's the [std::swap](https://en.cppreference.com/w/cpp/algorithm/swap) too :P